dev-docs/adding-data/infra/chef-repo/environments/_default.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "_default",
```

Comment:
I don't recall exactly, but I don't think you need to/want to make the `_default` environment yourself.